### PR TITLE
Update build instructions for Windows

### PR DESCRIPTION
**Pull request by ResearchDaniel**
Added a step for users who are using Anaconda for Python.

### CODE REVIEW

File: pages/manual/manual-gettingstarted-build.md [hunk 1]
```diff
@@ -23,20 +23,25 @@ You will need at least
 1. `git clone --recurse-submodules https://github.com/inviwo/inviwo`
 The `--recurse-submodules` is necessary to pull dependencies.
 
-2. Open CMake (we recommend using the GUI here), enter the source path and the preferred build directory (outside the inviwo directory!) and hit configure. You can then select the desired Inviwo modules (`IVW_MODULE_*`) and configure again. When selecting the compiler, make sure to select the correct Visual Studio version that you use on 64-bit. 32-bit is not supported.
-3. (Optional) To add external Inviwo modules, add those in `IVW_EXTERNAL_MODULES` in the format of
+2. *Only if you are using Anaconda for your Python environment:*
+   - Add an environment variable `CMAKE_PREFIX_PATH` and set it to your Qt dir, e.g., `Qt/5.12.1/msvc2017_64` - This will ensure that CMake finds *your* Qt instead of its own.
```
Comment (petersteneteg) [inline pages/manual/manual-gettingstarted-build.md]: Cmake doesn't have a qt install...

Comment (petersteneteg) [reply pages/manual/manual-gettingstarted-build.md]: Explain the you mean the one in anaconda...

Comment (ResearchDaniel) [reply pages/manual/manual-gettingstarted-build.md]: Done

**Comment by ResearchDaniel** [comment]
@xeTaiz Linux does not need the added instructions, right?

**Review by petersteneteg — APPROVED** [review]

**Review by martinfalk — APPROVED** [review]

**Comment by xeTaiz** [comment]
Sorry for being idle. I have added another note on the Anaconda on Linux situation. I'll merge this then :)

**Pull request merged**